### DISCUSSION TITLE
Content API Collection

### DISCUSSION THREAD

**Discussion by tomlongo** [statamic/cms#4334] [Troubleshooting]
I'm trying to experiment with the Content API but the tree endpoints are not returning anything, eg:

Endpoints: 
`/api/collections/pages/tree`
`/api/navs/primary/tree`

Response:
```
{
message: "Not found."
}
```

Some endpoints do work, however, for example:

- `/api/collections/pages/entries/xxxx-xxxx-xxxx` -- this one works ok
- `/api/globals` -- works OK

I may have configured things wrong. Is this the correct setup in `config/statamic/api.php`?

```
'resources' => [
        'collections' => true,
    ],
```
Not sure if that is right?

**Comment by duncanmcclean** [comment]
Have you enabled `navs` too? 

https://github.com/statamic/statamic/blob/master/config/statamic/api.php#L22

**Reply by tomlongo** [reply]
Actually no i hadnt enabled that. I've updated `config/statamic/api.php` so that it is now:

```
'resources' => [
        'collections' => true,
        'navs' => true,
    ],
```
But the `/api/collections` and `/api/navs` endpoints are still returning Not found. 

Interestingly though, even if i remove the `resources` line altogerher from `api.php` my entry endpoint still works (eg. `/api/collections/pages/entries/0e06a9c4-b2ee-4c3e-b28d-9be7c6c21991`)

shouldn't that stop working? Wondering if the `resources` setting is having any effect

**Reply by jasonvarga** [reply]
Do you have `'cache'` in your `api.php` file? Try clearing your cache with `php artisan cache:clear`, or changing that setting to `'cache' => false`.

**Reply by tomlongo** [reply]
hey mate thanks for the help. I've tried running `php artisan cache:clear` and adding cache false to the `api.php` but same result unfortunately!

I know the collection name `pages` is correct becuase the `/api/collection/pages/entries/...` endpoint works OK, it's just the `/tree` and nav ones that are having a hard time 🤔
